openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -261,11 +261,11 @@ def __init__(
         Load one SMIRNOFF parameter set in XML format (searching the package data directory by default, which includes
         some standard parameter sets):
 
-        >>> forcefield = ForceField('test_forcefields/test_forcefield.offxml')
+        >>> forcefield = ForceField('openff-2.0.0.offxml')
 
         Load multiple SMIRNOFF parameter sets:
 
-        >>> forcefield = ForceField('test_forcefields/test_forcefield.offxml', 'test_forcefields/tip3p.offxml')
+        >>> forcefield = ForceField('openff-2.0.0.offxml', 'ff14sb_off_impropers_0.0.3.offxml')
```

Comment:
(blocking) It's going to add several seconds to docstring CI to load the ff14sb port. Let's revert the second FF here to just load TIP3P from the test folder. This would be a good candidate for the water models, once those get merged into openff-forcefields. 
```suggestion
        >>> forcefield = ForceField('openff-2.0.0.offxml', 'test_forcefields/tip3p.offxml')
```